CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/value/operations/Contains.java

```diff
@@ -43,6 +47,33 @@ public boolean execute(String p1, String p2) {
     public static void register(Operators operators) {
         final OperatorRegistry registry = operators.getRegistry(NAME);
         STRING_OPERATION.register(registry);
+
+        // Used when query is as follows: x contains '1'
```

Comment:
This is a minor one but I would find it a bit easier to read if you start with "e.g" `// e.g. Used when ...` as it took me a little while to understand that you were providing examples here.